crowbar_framework/app/models/node_object.rb

```diff
@@ -1210,8 +1224,11 @@ def set_state(state)
       rescue Timeout::Error
         Rails.logger.warn("chef client seems to be still running after 5 minutes of wait; going on with the reboot")
       end
-
-      ssh_cmd("/sbin/reboot")
+      if @node[:platform] == "windows"
+        bmc_cmd("power cycle")
```

Comment:
@bkutil that would require extra checks for "power cycle" and "power off" because ssh_cmd will not longer apply, so having it in bmc is better.